website/content/en/docs/building-operators/helm/quickstart.md

```diff
@@ -3,321 +3,102 @@ title: Helm Based Operator Quickstart
 linkTitle: Quickstart
 weight: 2
 ---
+## Prerequisites
```

Comment:
This section can be removed now that the main installation doc lists docker and kubectl.